src/DweetCard.tsx

```diff
@@ -497,7 +497,13 @@ export const DweetCard: React.FC<Props> = (props) => {
                 : 'none',
           }}
         >
-          <Linkify>{dweet.comments[0].text}</Linkify>
+          <Linkify componentDecorator={(decoratedHref, decoratedText, key) => (
+            <a target="blank" href={decoratedHref} key={key}>
```

Comment:
For some reason I can't get this to work locally, inspecting the <a> element doesn't show a target property at all :|

Also, this <Linkify> is only for the "title" comment at the top of the dweet. I assume you'd want to add this to all comments as well, which you can do from the Linkify tag at line 603